click_odoo_contrib/initdb.py

```diff
@@ -35,7 +35,21 @@ def check_cache_prefix(cache_prefix):
             "Invalid cache prefix name '{}'".format(cache_prefix))
 
 
-def odoo_createdb(dbname, demo, module_names):
+def _patch_ir_attachment_store():
+
+    def _db_storage(self):
+        return 'db'
+
+    from odoo.addons.base.ir.ir_attachment import IrAttachment
```

Comment:
@sbidoul I get an error here (9.0). And it works if i replace `odoo` by `openerp`.

```
ImportError: No module named addons.base.ir.ir_attachment
```